docs/src/modules/java-protobuf/partials/query-syntax-advanced.adoc

```diff
@@ -314,7 +314,7 @@ include::java:example$java-spring-view-store/src/it/java/store/view/TestKitConfi
 ----
 // tag::grpc[]
 
-For running a local proxy in Docker, with advanced view features enabled, use the environment variable `VIEW_FEATURES_ALL=on`.
+For running a local proxy in Docker, with advanced view features enabled, set the environment variable `VIEW_FEATURES_ALL: "true"`.
```

Comment:
Should we also added it to the template, default to `false` with a message telling users to contact us about enabling it in prod? 

It will make is easier for users if they can immediately see what and where to enable it in the docker-compose file.

Reply:
Yes, I think we should add that to the template.

Reply:
Ok, I will merge this one and send another PR for the templates and samples. I noticed some other issues there as well.